blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -200,14 +204,20 @@ def make_device_icon(self, icon_info: Gtk.IconInfo, is_paired: bool = False, is_
         ctx = cairo.Context(target)
 
         if is_paired:
-            icon_info = self.get_icon_info("dialog-password", 16, False)
+            _icon_info = self.get_icon_info("dialog-password", 16, False)
```

Comment:
What is the point of the new variable? Below does the same, right?

```python
icon_info = self.get_icon_info("dialog-password", 16, False)
assert icon_info is not None
```

Reply:
`icon_info` is already defined as non-optional in the method arguments, so we cannot assign the optional value from `get_icon_info` to it, but I could actually save `icon_info = _icon_info` as I could just use `_icon_info.load_surface` after the assertion directly.